docs/UserGuide.md

```diff
@@ -98,21 +98,16 @@ Adds a patient to the address book.
 
 Format: `add id/PATIENT_HOSPITAL_ID n/NAME p/PREFERRED_NAME f/FOOD_PREFERENCE+ c/FAMILY_CONDITION+ h/HOBBY+ [t/TAG]…​`
 
-<box type="tip" seamless>
-
-**Tip:** 
 * The `add` command accepts parameters which consists of:
-    *  `patientHospitalId` integer,
-    * `name`, `preferredName` String with only alphabets character,
+    *  `patientHospitalId` non-negative numeric characters which uniquely identifies the patient,
+    * `name`, `preferredName` String contains alphanumeric characters,
     *  `foodPreference`, `familyCondition`, `hobby` String and all kinds of characters, 
     *  `tag` which are alphanumeric.
 * A patient can have any number of tags (including 0).
 * A patient can have more than 1 `f/FOOD_PREFERENCE`, `c/FAMILY_CONDITION` and `h/HOBBY`.
```

Comment:
change from 1 to one or  a single to standardise with the rest of the DG, where 1 would be used for indexing or examples, and when talking about the explanation, use one instead of confusion

Reply:
agree!